manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PascalABCNET.PascalABCNET-stable
+PackageVersion: 3.8.2.3074
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: http://pascalabc.net/downloads/PascalABCNETMiniSetup.exe
+  InstallerSha256: FCE97894999A183C07AD845B770952ABBAD9BE501B84AD7E3A17CD72DC0B57E3

```

```suggestion
  InstallerSha256: FCE97894999A183C07AD845B770952ABBAD9BE501B84AD7E3A17CD72DC0B57E3
  ProductCode: PascalABCNET
  AppsAndFeaturesEntries: 
    - DisplayName: PascalABC.NET
      DisplayVersion: null
      Publisher: null
      ProductCode: PascalABCNET
```
ProductCode will be needed for package matching

---

What is DisplayVersion and why is it set to null? Is ProductCode used for the Match field?

---

DisplayVersion is the version shown in Control Panel on Windows 10